java/README.md

```diff
@@ -10,24 +10,24 @@ $ ./gradlew installDist
 Run the account server:
 
 ```
-$ ./build/install/wallet/bin/account-server
+$ ./build/install/wallet/bin/account-server [--creds=insecure|xds]
```

Comment:
Can this just be left out? There are a number of optional flags. This stops copy-paste from working and, without exposition, doesn't add anything to the content in the readme.

Reply:
Okay, I will revert my changes to this file. We can separately document all the options.